src/bundle/Resources/public/scss/ui/modules/universal-discovery/_collapsible.scss

```diff
@@ -0,0 +1,100 @@
+.c-collapsible {
+    $self: &;
+
+    & + .c-filters-row,
+    & + .c-filters__row {
```

Comment:
something is wrong with this for me :) `c-filters-row` and `c-filters__row` should be the same component but now it's separate or am I wrong

Reply:
I created some new filters components and now we have `c-filters-row` and we no longer have `c-filters__row`, but because this is an add-on for 4.6 I wanted to be extra careful with BC break and didn't delete this CSS styles. But maybe I should not be that careful. :)